kolla/node_custom_config/zun/policy.yaml

```diff
@@ -2,3 +2,5 @@
 container:create:runtime: rule:admin_or_owner
 container:create:requested_destination: rule:admin_or_owner
 container:delete_force: rule:admin_or_owner
+host:get: "@"
```

Comment:
Bit of a weird syntax, but OK! Maybe should have a comment "@ = allow all" above?

Reply:
Yes, there are a few ways to allow all, the other two looked a bit too much like "allow none" for me. They are:
[] and ""

https://docs.openstack.org/oslo.policy/latest/admin/policy-yaml-file.html#syntax